src/AppBundle/Entity/TaskCollection.php

```diff
@@ -58,25 +58,65 @@ public function getItems()
         return $this->items;
     }
 
+    public function getMaxPosition() {
+        $maxPosition = -1;
+        foreach ($this->getItems() as $i) {
+            $maxPosition = $i->getPosition() > $maxPosition ? $i->getPosition() : $maxPosition;
+        }
+
+        return $maxPosition;
+    }
+
     public function addTask(Task $task, $position = null)
```

Comment:
Ok, I'll review tomorrow. 
At the beginning it was managed by hand, but I thought it would be easier to use the behavior, my bad 😐
I think it is the `Sortable` behavior which is buggy and unmaintained, this library is kind of a reference for other things.

Reply:
[ref](https://github.com/Atlantic18/DoctrineExtensions/issues?utf8=%E2%9C%93&q=is%3Aopen+label%3Abug+sortable)